packages/cli/src/config.ts

```diff
@@ -53,6 +53,7 @@ export const jsonToGlobalConfig = (json: unknown) => {
 export type GlobalConfig = z.infer<typeof zGlobalConfig>;
 
 export const PROJECT_TEMPALTES = [
+  "vanilla",
```

Comment:
Added new `vanilla` option to export as plain remix projects without any deploy target defaults.